content/logs/aws.md

```diff
@@ -0,0 +1,122 @@
+---
+title: AWS Log Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+beta: true
+---
+
+## Overview
+
+Push your logs from ELB, S3, CloudTrail, VPC, CloudFront and CloudWatch into an S3 bucket bind to a lambda function. 
+Using a Lambda function, called whenever a new S3 log file is made available, avoids the hassle of creating processes that would pull periodically incoming S3 logs.
+
+## Setup
+### Installation
+#### Create a new lambda function
+
+1. Navigate to the [Lambda Console](https://console.aws.amazon.com/lambda/home?region=us-east-1) and create a new function:
+    {{< img src="logs/aws/create_lambda_function.png" alt="Create Lambda function" >}}
+
+2. Select **Author from scratch** and give the function a unique name.
+    {{< img src="logs/aws/author_from_scratch.png" alt="Author from Scratch" >}}
+3. For `Role`, select **Create new role from template(s)** and give the role a unique name
+4. Under Policy templates, search for and select **s3 object read-only permissions**
+    {{< img src="logs/aws/basic_information.png" alt="Basic Information" >}}
+5. Select **Create Function**
+
+#### Provide the code and configure the lambda
+
+6. Copy and paste the code from [This repo](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py) into the function code area.
+7. Change the Runtime to **Python 2.7**
+    {{< img src="logs/aws/select_python.png" alt="Select Python" >}}
```

Comment:
possible to get a pic that doesnt wrap the name in such an ugly way?

Reply:
ping @NBParis